packages/ui-library/src/components/checkbox/index.test.ts

```diff
@@ -34,6 +34,20 @@ describe('blr-checkbox', () => {
     expect(className).to.contain('input-control');
   });
 
+  it(' error Icon is set to undefined', async () => {
+    const element = await fixture(
+      BlrCheckboxRenderFunction({
+        ...sampleParams,
+        errorIcon: undefined,
+      })
+    );
+
+    const checkBox = querySelectorDeep('textarea', element.getRootNode() as HTMLElement);
+    const errorIcon = checkBox?.getAttribute('errorIcon');
+
+    expect(errorIcon).to.be.equal(undefined);
```

Comment:
Could you may be check for undefined directly i.e. expect(errorIcon).beUndefined